blog/content/post/2022-12-16-trenchboot-aem-for-qubesos.md

```diff
@@ -0,0 +1,358 @@
+favorite operating system. We're excited to share our plans to integrate the
+TrenchBoot Project into Qubes OS's new Anti-Evil Maid (AEM) implementation. As
+you may know, traditional firmware security measures like UEFI Secure Boot and
+measured boot, even with a Static Root of Trust (SRT), may only sometimes be
+enough to ensure a completely secure environment for your operating system.
+Compromised firmware may allow for the injection of malicious software into
+your system, making it difficult to detect. To overcome these limitations, many
+silicon vendors have started implementing Dynamic Root of Trust (DRT)
+technologies to establish a secure environment for operating system launch and
+integrity measurements. We're excited to take advantage of these advancements
+through integration with the [TrenchBoot Project]((https://trenchboot.org/)).
+
+The usage of DRT technologies like Intel Trusted Execution Technology (TXT) or
+AMD Secure Startup is becoming more and more significant; for example, Dynamic
+Root of Trust for Measurement (DRTM) requirements of [Microsoft Secured Core PCs](https://docs.microsoft.com/en-us/windows-hardware/design/device-experiences/oem-highly-secure#what-makes-a-secured-core-pc).
+DRTM has yet to find its place in open-source projects, but that gradually
+changes. The demand for having firmware-independent Roots of Trust is
+increasing, and projects that satisfy this demand are growing TrenchBoot is a
+framework that allows individuals and projects to build security engines to
+perform launch integrity actions for their systems. The framework builds upon
+Boot Integrity Technologies (BITs) that establish one or more Roots of Trust
+(RoT) from which a degree of confidence that integrity actions were not
+subverted.
+
+[Qubes OS Anti Evil Maid(AEM)](https://blog.invisiblethings.org/2011/09/07/anti-evil-maid.html)
+software heavily depends on the availability of DRTM technologies to prevent
+Evil Maid attacks. However, the project hasn't evolved much since the beginning
+of 2018 and froze on the support of TPM 1.2 with Intel TXT in legacy boot mode
+(BIOS). Because of that, the usage of this security software is effectively
+limited to older Intel machines only. TPM 1.2 implemented SHA1 hashing
+algorithm, which is nowadays considered weak in the era of forever-increasing
+computer performance and quantum computing. The solution to this problem comes
+with a newer TPM 2.0 with more agile cryptographic algorithms and SHA256
+implementation by default.
+
+The post will present the TrenchBoot solution for Qubes OS AEM replacing the
+current TPM 1.2 and Intel TXT-only implementation. The advantage of TrenchBoot
+solution over existing [Trusted Boot](https://sourceforge.net/p/tboot/wiki/Home/)
+is the easier future integration of AMD platform support, as well as TPM 2.0
+and UEFI mode support.
+
+# Modificationts to original Qubes OS AEM
+
+To replace the original implementation of Qubes OS AEm based on Trusted Boot,
```

Comment:
```suggestion
To replace the original implementation of Qubes OS AEM based on Trusted Boot,
```

Reply:
Resolved in: https://github.com/3mdeb/news-and-ideas/pull/503/commits/51d8b96d397437da8715495891dbe550493ad3f4